src/dbus/properties.cpp

```diff
@@ -246,8 +248,17 @@ void DBusPropertyGroup::requestPropertyUpdate(DBusPropertyCore* property) {
 		const QDBusPendingReply<QDBusVariant> reply = *call;
 
 		if (reply.isError()) {
-			qCWarning(logDbusProperties).noquote() << "Error updating property" << propStr;
-			qCWarning(logDbusProperties) << reply.error();
+			const auto& error = reply.error();
+			// For optional properties, suppress "No such property" errors specifically
+			if (!property->isRequired() && error.type() == QDBusError::InvalidArgs
+			    && error.message().contains("No such property"))
+			{
+				qCDebug(logDbusProperties).noquote() << "Error updating optional property" << propStr;
+				qCDebug(logDbusProperties) << error;
+			} else {
+				qCWarning(logDbusProperties).noquote() << "Error updating property" << propStr;
+				qCWarning(logDbusProperties) << error;
+			}
 		} else {
```

Comment:
Please leave this off, it shouldn't be part of an RSSI pr, especially because a direct property update to rssi is never requested.

Reply:
undid all changes to this file